tools/deversion/CMakeLists.txt

```diff
@@ -4,5 +4,4 @@
 add_executable(deversion main.c)
 
 target_link_libraries(deversion PRIVATE
-    k4ainternal::dynlib
-    k4ainternal::logging)
+    k4ainternal::dynlib)
```

Comment:
Doesn't deversion depend on logging? I realize dynlib is pulling it in, but we are explicitly doing things in main.c for logging.

Reply:
I struggled on this one because I didn't see deversion using logging. So I removed the lib. So I attempted to address the issue with the comment in deversion\main.c